gensim/models/wrappers/ldamallet.py

```diff
@@ -199,10 +199,10 @@ def load_word_topics(self):
                 if token not in word2id:
                     continue
                 tokenid = word2id[token]
-                wordtopics[int(topic), tokenid] += 1.0
-        logger.info("loaded assigned topics for %i tokens", wordtopics.sum())
-        self.wordtopics = wordtopics
+                word_topics[int(topic), tokenid] += 1.0
+        logger.info("loaded assigned topics for %i tokens", word_topics.sum())
         self.print_topics(15)
```

Comment:
You're right - the print was there in the code earlier and I removed the `self.wordtopics` assignment so now it doesn't make sense.